picard/util/__init__.py

```diff
@@ -1169,9 +1170,9 @@ def strxfrm(string):
 
 
 def detect_unicode_encoding(path):
```

Comment:
I'd renamed this to `detect_file_encoding` (because it no longer is just for Unicode)